.ruby-version

```diff
@@ -1 +1 @@
-1.9.3
+2.2.4
```

Comment:
how would you feel about either bumping this to 2.3 or drop the file altogether? i have no strong preference for having this file here in the first place, but *if* it's here, it should probably point to the most recent ruby version (that's being tested on travis too). that being said, i guess i'd rather remove it as it introduces maintenance overhead for comparatively little gain?